provisioner.go

```diff
@@ -330,17 +355,29 @@ type Disembedder func(src string) ([]byte, error)
 // NewDisembedderProvisioner creates a provisioner that can provision based on embedded data
 func NewDisembedderProvisioner(d Disembedder, pathAstilectron, pathElectron string, l astikit.StdLogger) Provisioner {
 	dp := &defaultProvisioner{l: astikit.AdaptStdLogger(l)}
-	dp.moverAstilectron = func(ctx context.Context, p Paths) (err error) {
+	dp.moverAstilectron = func(ctx context.Context, p Paths) (closeFunc func() error, err error) {
 		if err = Disembed(ctx, dp.l, d, pathAstilectron, p.AstilectronDownloadDst()); err != nil {
-			return fmt.Errorf("disembedding %s into %s failed: %w", pathAstilectron, p.AstilectronDownloadDst(), err)
+			return nil, fmt.Errorf("disembedding %s into %s failed: %w", pathAstilectron, p.AstilectronDownloadDst(), err)
 		}
-		return
+		return func() (err error) {
+			if err = os.Remove(p.AstilectronDownloadDst()); err != nil {
+				return fmt.Errorf("removing %s failed: %w", p.AstilectronDownloadDst(), err)
+			}
+			dp.l.Debugf("Remove download file: %s", p.AstilectronDownloadDst())
```

Comment:
Adding a debug log for this action makes sense. I would however 

1) place it before `os.Remove()`
2) replace the text with `removing %s`

Reply:
done. https://github.com/asticode/go-astilectron/pull/358/commits/3481a4e396ea2fa4cef5282d957edcc4dd50505b#diff-752145c98ffd12d3a2b292517f8cf139ca3bcaa6512dc88dd7d8628372fd5df4R363-L366